frontend/src/components/Tree/Tree.jsx

```diff
@@ -83,7 +85,15 @@ function renderTree(dimensions, jsonData, svgRef, onNodeClick) {
   nodeGroupEnter
     .append("circle")
     .merge(nodeGroup.select("circle"))
-    .attr("r", 4);
+    .attr(
+      "transform",
+      (d) => `
+        rotate(${(d.x * 180) / Math.PI - 90})
+        translate(${d.y},0)
+      `
+    )
+    .attr("fill", (d) => (d.children ? "#555" : "#999"))
```

Comment:
Idk if this is the best idea. It looks kind of weird in this image:
<img width="584" alt="Screen Shot 2021-02-13 at 1 34 03 AM" src="https://user-images.githubusercontent.com/16051552/107843754-99a4b980-6d9b-11eb-9cd3-b077d0c4bf12.png">

Even though the nodes after the root are all at the same level, they have different colors, since some of them are leaf nodes and some others are not. I feel like nodes at the same level should have the same color like this: 
<img width="574" alt="Screen Shot 2021-02-13 at 1 41 58 AM" src="https://user-images.githubusercontent.com/16051552/107843877-be4d6100-6d9c-11eb-89b0-41337846a283.png">

What do you think?

Reply:
100% agree; I've fixed this